### DISCUSSION TITLE
CNC size ?

### DISCUSSION THREAD

**Discussion by macandnews** [arkypita/LaserGRBL#1586] [General]
Hello,

I have done laser engravings on my RS-CNC32 with LaserGRBL before without a problem, but today I have an unsolvable problem for me. :(
I don't use this software often, but have made all updates to the latest version. I have tried uninstalling the software and reinstalling it, but it doesn't solve my problem.

I wanted to engrave an image of approximately 100 x 70 mm at a specific location in my part by entering an offset of 35 mm in X and 170 mm in Y, which is far below the capabilities of my RS-CNC32 which has a machinable surface of 420 x 450 mm, but when creating the gcode, I have the following error message while my CNC is not connected to the PC since I am saving the file on an SD card ! :

https://www.makerfr.com/forum/download/file.php?id=4367

If I ignore the error message and save the gcode anyway, I have an error on the CNC when I start engraving telling me that I have reached the limits !!

However, I have already burned an image of 370 x 420 mm without problem !!

I did not find anything in the settings of LaserGRBL to indicate the size of my CNC and I do not understand why I am having this problem today when it used to work great before! I haven't changed anything in the software, I just applied the updates when the software offers them to me.

What to do ?
Thank you ;)

**Comment by arkypita** [comment]
Table size is configured inside controller, not inside LaserGRBL.
You should change this in GRBL configuration $130 $131.

https://github.com/gnea/grbl/wiki/Grbl-v1.1-Configuration

**Reply by macandnews** [reply]
> Table size is configured inside controller, not inside LaserGRBL.
> You should change this in GRBL configuration $130 $131.
> 
> https://github.com/gnea/grbl/wiki/Grbl-v1.1-Configuration

This is what I understood, but how is it that before I could burn images of 370 x 420 mm and that today, it does not work anymore although I have not changed anything, just did the software updates.

For that, I will have to connect my CNC to the PC because for the moment, I cannot modify anything?

Thank you


Sorry for my English, I am using google translate.

**Reply by arkypita** [reply]
LaserGRBL does not change controller setup by himself. Updates don't change anything either.

If it worked before and now it doesn't work, there are 3 options (in order of probability):
1) you changed something without knowing what you were doing
2) you used software other than LaserGRBL that changed the controller configuration
3) the controller configuration got corrupted on its behalf (sometimes it happens)

You can get everything back up and running simply by changing the controller configuration (Grbl Configuration).
[$20, $21, $130, $131](https://github.com/gnea/grbl/wiki/Grbl-v1.1-Configuration) are the involved parameters.

**Comment by macandnews** [comment]
Hello,
I am on a Mac using a Windows 10 Virtual PC with Parallels Desktop.
I don't know why, but can't have a COM port in LaserGRBL to connect to the CNC! :(

I copied my virtual PC to my MacBook Pro and there I managed to connect to the CNC and import the settings into LaserGRBL which works fine on my laptop.

Only, I cannot import the parameters into LaserGRBL on my main Mac because you have to be connected to the CNC to be able to import data.
I couldn't find a file on the PC to copy my settings to. :(

If you have an idea....
Thank you   ;)

Sorry for my English, I am using google translate.

**Comment by macandnews** [comment]
Hello,

Does anyone know where is the LaserGRBL file which contains the GRBL parameters of the CNC and can it be edited with Notepad or other?

Thank you.


Sorry for my English, I am using google translate.
